source/server-discovery-and-monitoring/server-discovery-and-monitoring-summary.rst

```diff
@@ -221,3 +216,8 @@ and single-threaded clients MUST request a scan before the next operation.
 The client MUST clear its connection pool for the server if the
 server is 4.0 or earlier, and MUST NOT clear its connection pool for the
 server if the server is 4.2 or later.
+
+Changelog
+---------
```

Comment:
Formatting is inconsistent with other files

```suggestion
=========
```

Reply:
This is intentional, as the RST file uses `---` as its top-level heading syntax.